### PR TITLE
[SG-1243] Location & Addresses

### PR DESCRIPTION
**Pull request by jacine**
Combines: #697, #716, and #717.

### CODE REVIEW

**Comment by jacine** [comment]
Re: https://github.com/SFDigitalServices/sfgov/pull/727/commits/639ea0f07be1645ae1c0a8ae9d3a85a5988b43f3  Sorry about that. Thank you @tregismoreira!

**Review by arlina-espinoza — APPROVED** [review]
Good work! All looks good, but I'd just prioritize enabling translations for the location content type.

File: web/modules/custom/sfgov_locations/sfgov_locations.module [hunk 1]
```diff
@@ -0,0 +1,311 @@
...
+      // Query list of Departments, sorted by Title.
+      $storage = Drupal::getContainer()->get('entity_type.manager')->getStorage('node');
+      $nids = $storage->getQuery();
+      $nids = $nids->condition('type', 'department')
+        ->condition('status', 1)
+        ->sort('title')
+        ->execute();
+
+      if (!$nids) {
+        return FALSE;
```
[the remark this answers is not in the document]
Comment (jacine) [reply web/modules/custom/sfgov_locations/sfgov_locations.module]: Good catch! Just pushed a fix! Thanks so much @arlina-espinoza.

**Review by arlina-espinoza — APPROVED** [review]
LGTM!

**Review by aekong — APPROVED** [review]

**Pull request merged**